app/helper.R

```diff
@@ -137,30 +134,62 @@ print_jco_manuscript <- function() {
   trmManuscript <- a("Prediction of Treatment-Related Mortality after Induction Therapy for Newly Diagnosed Acute Myeloid Leukemia",
                      href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3221524/")
   
+  manuscript_one <- a("Second Allogeneic Hematopoietic Cell Transplantation for Relapsed Adult Acute Myeloid Leukemia: Outcomes and Prognostic Factors",
+                     href="https://www.sciencedirect.com/science/article/pii/S2666636724004810")
+  
+  manuscript_two <- a("Utility of the Treatment-Related Mortality (TRM) score to predict outcomes of adults with acute myeloid leukemia undergoing allogeneic hematopoietic cell transplantation",
+                     href="https://www.nature.com/articles/s41375-022-01574-5")
```

Comment:
Use the DOI link for manuscripts. For example, https://doi.org/10.1038/s41375-022-01574-5